jet/dashboard/dashboard.py

```diff
@@ -7,7 +7,7 @@
 from django.template.loader import render_to_string
 from jet.dashboard import modules
 from jet.dashboard.models import UserDashboardModule
-from django.utils.translation import ugettext_lazy as _
+from django.utils.translation import gettext_lazy as _
```

Comment:
`ugettext(), ugettext_lazy(), ugettext_noop(), ungettext(), and ungettext_lazy()` are deprecated in favor of the functions that they’re aliases for: `gettext(), gettext_lazy(), gettext_noop(), ngettext(), and ngettext_lazy()`.